### PR TITLE
add jwks usage to readme

### LINKED ISSUE #132: Document how to support JWKS
## 🚀 Feature Proposal

Support for JWKS

## Motivation

Many services (e.g. AWS Cognito, Auth0) support JWKS and this plugin should be able to verify/decode tokens generated by those services.

We are introducing support for JWKS in [fast-jwt](https://github.com/nearform/fast-jwt/issues/63) via https://github.com/nearform/jwks-fetch, which has been extracted from https://github.com/nearform/fastify-auth0-verify. In the same way, we could support JWKS in this package.

### PR DESCRIPTION
**Pull request by conor909**
Closes #132

Updated the readme to include usage with verifying with JWKS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -543,6 +543,39 @@ fastify.listen(3000, function (err) {
 })
 ```
 
+### Varifying with JWKS
```
Comment (simoneb) [inline README.md:546]: typo

File: README.md [hunk 2]
```diff
@@ -543,6 +543,39 @@ fastify.listen(3000, function (err) {
...
+
+The following example integrates the [get-jwks](https://github.com/nearform/get-jwks) package to fetch a JWKS and varify a JWT against a valid public JWK.
```
Comment (simoneb) [inline README.md:548]: typo

**Review by simoneb — APPROVED** [review]
two typos, lgtm otherwise

File: README.md (hunk 2 above)
Comment (conor909) [reply README.md]: Thank you!

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**